### PR TITLE
179 Update Video Asset

### LINKED ISSUE #179: Update video asset

### PR DESCRIPTION
**Pull request by ichung08**
Created new guide on how to update video asset

Resolves #179

### CODE REVIEW

File: docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: dashboard
+title: Dashboard
+metaTitle: Store video asset on IPFS
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx:6]: A couple frontmatter fixes:
- [ ]  We can remove the `metaTitle` frontmatter field. This isn't actually a field docusaurus recognizes.
- [ ] Let's add `pagination_next: guides/some-guide` so it links to an appropriate next guide at the end of the doc. Right now it links to the api tab.

File: docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Step 3: Update an asset
+Click on `Edit Asset` to update an asset.
+
+![Update asset](/../static/guides/Dashboard-7.png)
+
+Here you can update the asset's display name and metadata.
+
+![Edit asset](/../static/guides/Dashboard-8.png)
+
+![Updated asset](/../static/guides/Dashboard-9.png)
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx:38]: Can we add a spotlight on the updated meta data as well?

File: docs/04-guides/on-demand/update-video-asset/02-api.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+```js
+const storeAssetOnIPFS = await fetch("https://livepeer.studio/api/asset/{id}", {
+    method: "PATCH",
+    headers: {
+      Authorization: `Bearer ${process.env.API_TOKEN}`,
+      "Content-Type": "application/json",
+    },
+    body: {
+        "name": "Example name"
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/update-video-asset/02-api.mdx:61]: Can we include an example code snippet on how to update metadata in addition to the name?

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx (hunk 1 above)
Comment (ichung08) [reply docs/04-guides/on-demand/update-video-asset/01-dashboard.mdx:6]: I'll add pagination_next for now but I think we'll have to adjust this later as the other guides for on demand get added

File: docs/04-guides/on-demand/update-video-asset/02-api.mdx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them 
+ready for optimized playback. Videos can be provided either by you (static content) 
+or your users, given your application offers an interface for them to do so. 
+
+You should be familiar with [Assets](/reference/api/get-assets) and [Tasks](/reference/api/get-tasks) 
+and have generated a [Livepeer Studio API key](/quickstart/).
+
+## Step 1: Get the `asset.id` of an existing asset
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/update-video-asset/02-api.mdx:22]: We should not advise a developer to list all assets just to find one. There could theoretically be thousands. We should assume they already have the asset id in their own database and are using it to make a request to get more info about it.

File: docs/04-guides/on-demand/update-video-asset/02-api.mdx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+```bash
+curl --location --request GET 'https://livepeer.studio/api/asset' \
+--header 'Authorization: Bearer $API_TOKEN'
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
+## Step 2: Update the asset
+Once you have an `asset.id`, you can make a request to update the asset's storage location.
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/update-video-asset/02-api.mdx:51]: The code example is showing up to update an asset's name and metadata not its location.

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: docs/04-guides/on-demand/update-video-asset/02-api.mdx (hunk 4 above)
Comment (ichung08) [reply docs/04-guides/on-demand/update-video-asset/02-api.mdx:22]: If that's the case and they already have an `asset.id` wouldn't they just jump to step 2?

Comment (adamsoffer) [reply docs/04-guides/on-demand/update-video-asset/02-api.mdx:22]: Yup. You can remove step 1.

**Review by adamsoffer — APPROVED** [review]

**Pull request merged**